unity-client/Assets/Scripts/Tests/TestsBase.cs

```diff
@@ -63,8 +63,7 @@ protected virtual IEnumerator TearDown()
 
         TestHelpers.ForceUnloadAllScenes(SceneController.i);
 
-        if (PointerEventsController.i != null)
-            PointerEventsController.i.Cleanup();
+        Environment.i.Cleanup();
```

Comment:
Just in case: this would clean all the `Environment` systems, not only the `PointerEventsController`. Make sure this change isn't breaking something else.

Reply:
Agree, but I think it's better to clear up all the environment after each test. One test should not rely on previous executions.

Reply:
Of course, I just let this comment in case we are already cleaning something in `TearDown` that we are already cleaning here.